webapp/channels/src/components/suggestion/at_mention_provider/at_mention_provider.tsx

```diff
@@ -458,3 +460,26 @@ export default class AtMentionProvider extends Provider {
         };
     }
 }
+
+defineMessages({
```

Comment:
For whatever reason, `t` was called for a bunch of suggestion-related strings in `utils/constants` which you'll see below. I took some time to figure out which strings were used where and moved them to their corresponding providers.

Eventually, I'd like to change suggestions/providers so that the provider explicitly adds dividers to a list of items because I don't like how we construct the IDs for these translation strings or how dividers are sort of implied by the suggestion data. I started a bit on that before realizing that that was way out of scope for what I wanted to do here